api/src/main/java/io/minio/UploadObjectArgs.java

```diff
@@ -64,27 +64,27 @@ private void validateFilename(String filename) {
       }
     }
 
-    public Builder filename(String filename) throws IOException {
+    public Builder filename(String filename, long partSize) throws IOException {
```

Comment:
Minor refactoring suggestion: Calls to `validateSizes` and `validatePartCount` can be moved inside `partInfo`

Reply:
Done